src/components/students-in-categories/StudentsInCategoriesChart.tsx

```diff
@@ -0,0 +1,64 @@
+import { Doughnut } from 'react-chartjs-2'
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js'
+import { Box, Typography } from '@mui/material'
+import { styles } from './StudentsInCategories.styles'
+import { useTranslation } from 'react-i18next'
+import { categories } from './StudentsInCategories.constants'
+
+ChartJS.register(ArcElement, Tooltip, Legend)
+
```

Comment:
please move `data` and `options` to a file with constants, like `StudentsInCategoriesChart.constants.tsx`, and import them in this file